website/docs/core-concepts/connecting-to-data-sources/authentication/connect-to-apis.md

```diff
@@ -4,24 +4,102 @@ description: You can hit any REST endpoint available on the public internet
 
 # Configuring APIs
 
+Configuring APIs involves setting up and managing the various aspects of an API that allow it to function effectively. This can include defining the API's structure and capability, setting up authentication and authorization, and establishing connections with other systems or services.
+
 :::tip
-If your API is not available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
+If you want to access an API that's not publicly available on the internet, you can use a service like [ngrok](https://ngrok.com/) to expose it. For more information, see [How to work with local APIs on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
 :::
 
-## API Editor
+## API query editor
```

Comment:
Can we replace `API query editor` with `REST API query editor` - since API can mean any protocol (non REST) as well. Similar for other usage below as well.

Reply:
FYI: Changed this back to API pane as in the current docs. not sure if we change it to query editor.